sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/TransformWithStateVariableUtils.scala

```diff
@@ -43,6 +43,10 @@ object TransformWithStateVariableUtils {
   def getMapState(stateName: String, ttlEnabled: Boolean): TransformWithStateVariableInfo = {
     TransformWithStateVariableInfo(stateName, StateVariableType.MapState, ttlEnabled)
   }
+
+  def getTimerState(stateName: String): TransformWithStateVariableInfo = {
+    TransformWithStateVariableInfo(stateName, StateVariableType.ValueState, ttlEnabled = false)
```

Comment:
Do we want this to be ValueState? Why not create another enum?

Reply:
Yea thats an option. However, kept it like this so that we can reuse the code on the reader side. But if we feel that we want to use an updated schema for timer anyway, then I can change this to a concrete type as well

Reply:
cc - @HeartSaVioR - thoughts ?

Reply:
Added this